minidump-processor/json-schema.md

```diff
@@ -177,6 +186,19 @@ anyway.
         "source_register": <string>
       }
     ],
+    
+    /// A list of inconsistencies detected by comparing crash reason/crash address with 
+    /// crashing instruction and memory information
+    "crash_inconsistencies": [
+      {
+        "inconsistency": "int_div_by_zero_not_possible"
+          | "priv_instruction_crash_without_priv_instruction"
+          | "non_canonical_address_falsely_reported"
+          | "access_violation_when_access_allowed"
+          | "crashing_access_not_found_in_memory_accesses"
+      }
```

Comment:
Hmm originally I thought it might also output other information like the access direction of the instruction vs the access direction of the crash, but it's shown elsewhere so it's not really necessary. Now that you mention it I can probably just leave this as a list of strings.